app/src/main/java/com/hamdan/forzenbook/data/repository/LoginRepository.kt

```diff
@@ -1,9 +1,9 @@
 package com.hamdan.forzenbook.data.repository
 
-import com.hamdan.forzenbook.data.network.LoginResponse
-
 interface LoginRepository {
     suspend fun requestValidation(email: String)
 
-    suspend fun getToken(email: String, code: String): LoginResponse?
+    suspend fun getTokenFromNetwork(email: String, code: String): LoginData?
```

Comment:
The repo should decide in one method where to get the token from, either network or database, like the other examples we have done

Reply:
combined methods into one essentialy